README.md

```diff
@@ -79,9 +79,9 @@ Building on Windows with Visual Studio requires version 2013 or higher.
 
 **Typical workflow:**
 
-0. Open an image and get an image object.
-0. Manipulate it.
-0. Save to disk / Send image buffer over network / etc.
+1. Open an image and get an image object.
```

Comment:
I used 0's on purpose. Markdown transforms a list of zeros to a numbered list automatically.